### PR TITLE
OHM-221: Reruns on basic filter - default to no reruns

### PR DESCRIPTION
**Pull request by BMartinos**
This PR relates to issue: #354 

Moved "rerun" filter to basic filters and showing no rerun transactions by default

### CODE REVIEW

**Comment by BMartinos** [comment]
Hi @rcrichton Would you mind giving this PR a review for me when you get a chance. Thanks

File: app/views/partials/transaction-filter-settings.html [hunk 1]
```diff
@@ -289,6 +279,19 @@
         <option value="500">500</option>
       </select>
     </div>
+    <div class="settings_div" style="float: right;">
+      <label>
+        Show Reruns: 
+        <i ng-show="uiSettings.showTooltips" tooltip-placement="top" tooltip-html-unsafe="Set whether to only show transactions that have been rerun" class="glyphicon glyphicon-question-sign tooltip-wide"></i>
```
Comment (rcrichton) [inline app/views/partials/transaction-filter-settings.html]: Perhaps delete the word 'only' from the tooltip text.

**Comment by rcrichton** [comment]
@BMartinos Looks good to me. Just one super minor comment above, otherwise :+1:

**Comment by BMartinos** [comment]
Great thanks @rcrichton. Will make that small correction and merge once done.

**Pull request merged**